recipes-support/rdmagent/rdmagent.bb

```diff
@@ -41,10 +41,15 @@ LDFLAGS:append = " -lsecure_wrapper"
 
 DEPENDS += "libsyswrapper"
 
-EXTRA_OECONF:append = " --enable-iarmbusSupport=yes --enable-t2api=yes"
+EXTRA_OECONF:append = " --enable-iarmbusSupport=yes --enable-t2api=yes --enable-mountutils=yes --enable-rdkcertselector=yes"
+
+DEPENDS:append = " ${@bb.utils.contains('DISTRO_FEATURES', 'safec', ' safec', " ", d)}"
+CFLAGS:append = " ${@bb.utils.contains('DISTRO_FEATURES', 'safec',  ' `pkg-config --cflags libsafec`', '-fPIC', d)}"
```

Comment:
-fPIC is already added in line 38 by default. Lets remove that. Also, bunch line 47 and 48